contracts/proxy-lib/src/ext_config.rs

```diff
@@ -0,0 +1,11 @@
+use near_sdk::ext_contract;
+use calimero_context_config::repr::Repr;
+use calimero_context_config::types::{SignerId, ContextId};
+
+#[ext_contract(config_contract)]
+pub trait ConfigContract {
```

Comment:
also, btw, you can use this directly from the crate if you added it as a dependency

```rust
use calimero_context_config_near::ContextConfigs;

let promise = ContextConfigs::ext(self.context_config_account_id)
    .with_static_gas(..)
    .members(context_id, offset, length)
    .then(..);
```